docs.html

```diff
@@ -0,0 +1,573 @@
+          <li class="nav-item">
+            <a
+              href="https://github.com/Ashish-j-shetty/fitstrap"
+              target="_blank"
+              class="nav__link"
+              >Github</a
+            >
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <nav class="mobile-nav">
+      <ul class="nav__list flex-col m-1">
+        <h2>Components</h2>
+        <li class="list__item mt-1">
+          <a class="menu__nav__link" href="#gettingStarted">Getting started</a>
+        </li>
+        <li class="lsit_item mt-1">
+          <a class="menu__nav__link" href="#avatar">Avatar</a>
+        </li>
+        <li class="lsit_item mt-1">
+          <a class="menu__nav__link" href="#alerts">Alert</a>
+        </li>
+        <li class="lsit_item mt-1">
+          <a class="menu__nav__link" href="#button">Button</a>
+        </li>
+        <li class="lsit_item mt-1">
+          <a class="menu__nav__link" href="#card">Card</a>
+        </li>
+        <li class="lsit_item mt-1">
+          <a class="menu__nav__link" href="#rating">Rating</a>
+        </li>
+      </ul>
+    </nav>
+
+    <main>
+      <div class="grid-col-2-spilt">
+        <div class="flex flex-col flex-start pt-1 desktop-nav p-6">
+          <nav>
+            <ul class="side__nav nav__list flex-col m-1">
+              <h2>Components</h2>
+              <li class="list__item mt-1">
+                <a class="menu__nav__link" href="">Getting started</a>
+              </li>
+              <li class="lsit_item mt-1">
+                <a class="menu__nav__link" href="#avatar">Avatar</a>
+              </li>
+              <li class="lsit_item mt-1">
+                <a class="menu__nav__link" href="#alerts">Alert</a>
+              </li>
+              <li class="lsit_item mt-1">
+                <a class="menu__nav__link" href="#button">Button</a>
+              </li>
+              <li class="lsit_item mt-1">
+                <a class="menu__nav__link" href="#card">Card</a>
+              </li>
+              <li class="lsit_item mt-1">
+                <a class="menu__nav__link" href="#rating">Rating</a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+        <div class="flex flex-col main">
+          <div id="gettingStarted" class="flex flex-col">
+            <h1>Getting Started</h1>
+            <p>
+              Use the following way to include Fitstrap styles and scripts into
+              your project
+            </p>
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%28171%2C184%2C195%2C0%29&t=seti&wt=none&l=htmlmixed&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=37px&ph=27px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%253Clink%2520%250A%2520%2520rel%253D%2522stylesheet%2522%2520%250A%2520%2520href%253D%2522https%253A%252F%252Ffitstrap.netlify.app%252Flib%252Fstyles.css%2522%2520%250A%2520%252F%253E"
+              style="
+                width: 699px;
```

Comment:
If you are using the same `IFRAME` then you can add styling by giving `class-name` so that you don't have to repeat yourself